### ISSUE TITLE
Data Files: Pagination buttons must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#574]
### Steps to recreate
Visit /data_files and run the Axe dev tools

### URL
http://localhost:3000/data_files

### Issues
Pagination buttons do not meet minimum color contrast ratio threshold

### Severity
High

### WCAG Success Criterion
- [ ] Elements must meet minimum color contrast ratio thresholds (2)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Some people cannot read faint text. For all consumers of visual content, adequate light-dark contrast is needed between the relative luminance of text and its background for good readability. 

### Recommended fix
To solve this problem, you need to fix the following:
- Element has insufficient color contrast of 3.56 (foreground color: #83888f, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved

### Notes
2 instances are the previous and next buttons
